diff --git a/node_modules/@wxperia/liquid-glass-vue/dist/index.js b/node_modules/@wxperia/liquid-glass-vue/dist/index.js
index 4f9848b..1e6d60f 100644
--- a/node_modules/@wxperia/liquid-glass-vue/dist/index.js
+++ b/node_modules/@wxperia/liquid-glass-vue/dist/index.js
@@ -24,7 +24,7 @@ class ShaderDisplacementGenerator {
     this.options = options;
     this.worker = new Worker(new URL(
       /* @vite-ignore */
-      "/assets/shader-worker-CJN-6C3l.js",
+      self.location.origin + "/assets/shader-worker-CJN-6C3l.js",
       import.meta.url
     ), {
       type: "module"
